vnext/src/index.windows.ts

```diff
@@ -8,7 +8,7 @@ export * from './Libraries/Components/DatePicker/DatePicker.windows';
 export * from './Libraries/Components/DatePicker/DatePickerProps';
 export * from './Libraries/Components/Flyout/Flyout.windows';
 export * from './Libraries/Components/Glyph/Glyph.windows';
-export * from './Libraries/Components/Picker/PickerWindows';
+export * from './Libraries/Components/Picker/Picker.windows';
```

Comment:
I think the some change is also needed in index.ts to import Picker?  and PickerWindows.tsx should be Picker.tsx?

Its possible @acoates-ms was intentionally trying to change the name to PickerWindows, seems it was partly done

Reply:
checked with coates, we can stick with 'Picker' for now since you have to import from 'react-native-windows' to get the 'windows-ness'

Reply:
Spoke offline about this. Apparently we are pulling a react-native "Picker" which is causing a collision if we rename PickerWindows.tsx to Picker.tsx
We decided to not make the recommended change.